docs/src/HowToGuides/BalanceLaws/how_to_define_bcs.md

```diff
@@ -0,0 +1,52 @@
+# How to define boundary conditions
```

Comment:
Should we also include the BC specification using the Dirichlet /  Neumann BCs in `boundary_state!` as is done in the Land Tutorial?

Reply:
Well, I was hoping to cover BC implementation in detail in the BC tutorial